CHANGES/1196.bugfix

```diff
@@ -0,0 +1 @@
+Implement new feature to remount /var/lib/pulp with the SELinux context (label.) See the new variable `pulp_selinux_remount_data_dir`'s description in pulp_common's README for more info.
```

Comment:
if it is a feature, shouldn't it be .feature?

Reply:
Yes, thank you!